rice-middleware/impl/src/test/java/org/kuali/rice/kew/rule/GenericAttributeContentTest.java

```diff
@@ -36,22 +36,22 @@
                                                   "      </field>" +
                                                   "    </boringAttribute>";
     private static final String ATTRIB2_CONTENT = "    <coolAttribute>" +
-                                                  "      <field>" +
-                                                  "        <name>car</name>" +
-                                                  "        <value>KIT</value>" +
-                                                  "      </field>" +
                                                   "      <field>" +
                                                   "        <name>driver</name>" +
                                                   "        <value>hasselhof</value>" +
                                                   "      </field>" +
+                                                  "      <field>" +
+                                                  "        <name>car</name>" +
+                                                  "        <value>KIT</value>" +
```

Comment:
Not saying it needs to be fixed since there are probably tests that rely on it being this way, but that is not the name of [the car in Knight Rider](https://en.wikipedia.org/wiki/KITT).

Reply:
Don't hassle the Hoff